modules/contact-form/css/gutenblock.css

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
Suppose this file is CSS instead of SCSS just to make building easier for now?

Let's switch it to SCSS as soon as we can so that we can rely on nesting, colour variables and other things like that so that as the CSS grows, we don't need to do too much refactoring.

Reply:
Yarp, that's the plan.